docs/UserGuide.md

```diff
@@ -2,16 +2,20 @@
 layout: page
 title: User Guide
 ---
+## Table of Contents
+{:.no_toc}
 
-FINDvisor is a **desktop app for financial advisors to manage contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, FINDvisor can get your contact management and meeting scheduling tasks done faster than traditional GUI apps.
-
-* Table of Contents
+* ToC
 {:toc}
 
 --------------------------------------------------------------------------------------------------------------------
 
 <div style="page-break-after: always;"></div>
 
+## About FINDvisor
+FINDvisor is a desktop app for **Singapore-based** financial advisors to manage contacts with **Singapore-based phone numbers**. If you **frequently schedule meetings and require quick access to your contacts' details**, then FINDvisor is the tool for you. It is optimized for use via a Command Line Interface ([CLI](https://aws.amazon.com/what-is/cli/#:~:text=A%20command%20line%20interface%20(CLI,all%20applications%20and%20software%20systems.))), suitable for individuals who prefer typing over mouse interactions.
```

Comment:
I am not sure about "optimized for use via a Command Line Interface"? I feel like that may not be the most appropriate? I think the emphasis regarding it being suitable for individuals who prefer to type over mouse interactions is sufficient?

Reply:
@KohGuanZeh I re-used this from the description in v1.3 UG, should we omit this instead?

Reply:
My only concern regarding this is that if we remove that line we will deviate from the original value proposition from the original addressbook3. I propose we keep it and add more CLI interactions under planned enhancements, if we have the capacity.

Also just a small nit, but Singapore-based mobile phone numbers should be specified since we had an issue regarding the use of Singapore home numbers.

Reply:
I think it there's less room for confusion if we say optimized for individuals who prefer to type.